gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageTestHelper.java

```diff
@@ -256,12 +264,23 @@ public TestBucketHelper(String bucketPrefix) {
 
     private static String makeBucketName(String prefix) {
       String username = System.getProperty("user.name", "unknown").replaceAll("[-.]", "");
-      username = username.substring(0, min(username.length(), 9));
+      // Only first 6 characters of username is considered.
+      username = username.substring(0, min(username.length(), 6));
       String uuidSuffix = UUID.randomUUID().toString().substring(0, 6);
-      return prefix + DELIMITER + username + DELIMITER + uuidSuffix;
+      // [6 char username] + DELIMITER + [6 char UUID] => 13 chars of unique Id
+      // Leaving room of 48 characters for bucketPrefix and suffix (excluding delimiters)
+      return prefix + username + DELIMITER + uuidSuffix;
```

Comment:
Let's keep in same order

Reply:
It is in same order. Its just that `prefix` already have DELIMITER at the end. So duplication is removed.